apps/preview/next/pages/showcases/Gallery/GalleryVertical.tsx

```diff
@@ -0,0 +1,148 @@
+  { imageSrc: Images.gallery8ThumbSrc, alt: 'backpack8' },
+  { imageSrc: Images.gallery9ThumbSrc, alt: 'backpack9' },
+  { imageSrc: Images.gallery10ThumbSrc, alt: 'backpack10' },
+];
+
+export default function GalleryVertical() {
+  const lastThumbRef = useRef<HTMLButtonElement>(null);
+  const thumbsRef = useRef<HTMLDivElement>(null);
+  const firstThumbRef = useRef<HTMLButtonElement>(null);
+  const [activeIndex, setActiveIndex] = useState(0);
+
+  const firstThumbVisible = useIntersection(firstThumbRef, {
+    root: thumbsRef.current,
+    rootMargin: '0px',
+    threshold: 1,
+  });
+
+  const lastThumbVisible = useIntersection(lastThumbRef, {
+    root: thumbsRef.current,
+    rootMargin: '0px',
+    threshold: 1,
+  });
+
+  const onDragged = (event: SfScrollableOnDragEndData) => {
+    if (event.swipeRight && activeIndex > 0) {
+      setActiveIndex((currentActiveIndex) => currentActiveIndex - 1);
+    } else if (event.swipeLeft && activeIndex < images.length - 1) {
+      setActiveIndex((currentActiveIndex) => currentActiveIndex + 1);
+    }
+  };
+
+  return (
+    <div className="relative max-h-[600px] flex h-full">
+      <SfScrollable
+        ref={thumbsRef}
+        className="items-center w-full [&::-webkit-scrollbar]:hidden [-ms-overflow-style:none] [scrollbar-width:none]"
+        direction="vertical"
+        activeIndex={activeIndex}
+        previousDisabled={activeIndex === 0}
+        nextDisabled={activeIndex === images.length - 1}
+        slotPreviousButton={
+          <SfButton
+            className={classNames(
+              'absolute !rounded-full z-10 top-4 rotate-90 bg-white',
+              firstThumbVisible?.isIntersecting ? 'hidden' : null,
+            )}
+            variant="secondary"
+            size="sm"
+            square
+            slotPrefix={<SfIconChevronLeft />}
+          />
+        }
+        slotNextButton={
+          <SfButton
+            className={classNames(
+              'absolute !rounded-full z-10 bottom-4 rotate-90 bg-white',
+              lastThumbVisible?.isIntersecting ? 'hidden' : null,
+            )}
+            variant="secondary"
+            size="sm"
+            square
+            slotPrefix={<SfIconChevronRight />}
+          />
+        }
+      >
+        {thumbImages.map(({ imageSrc, alt }, index, thumbsArray) => (
+          <button
+            // eslint-disable-next-line no-nested-ternary
+            ref={index === thumbsArray.length - 1 ? lastThumbRef : index === 0 ? firstThumbRef : null}
+            type="button"
+            aria-label={alt}
+            aria-current={activeIndex === index}
+            key={`${alt}-${index}-thumbnail`}
+            className={`md:w-[78px] md:h-auto relative shrink-0 pb-1 mx-4 border-b-4 snap-center cursor-pointer focus-visible:outline focus-visible:outline-offset transition-colors flex-grow md:flex-grow-0  ${
+              activeIndex === index ? 'border-primary-700' : 'border-transparent'
```

Comment:
Use `classNames` with this expressions:
```suggestion
              'border-primary-700' : activeIndex === index,
              'border-transparent': activeIndex !== index,
```